Mathlib/Algebra/Group/Defs.lean

```diff
@@ -954,29 +954,31 @@ variable [DivInvMonoid G] {a b : G}
 #align zpow_zero zpow_zero
 #align zero_zsmul zero_zsmul
 
-@[to_additive (attr := norm_cast) ofNat_zsmul]
-theorem zpow_ofNat (a : G) : ∀ n : ℕ, a ^ (n : ℤ) = a ^ n
+@[to_additive (attr := simp, norm_cast) coe_nat_zsmul]
+theorem zpow_coe_nat (a : G) : ∀ n : ℕ, a ^ (n : ℤ) = a ^ n
```

Comment:
While you're at it modifying almost all calls to those lemmas, could you change them to the proper name?
```suggestion
@[to_additive (attr := simp, norm_cast) natCast_zsmul]
theorem zpow_natCast (a : G) : ∀ n : ℕ, a ^ (n : ℤ) = a ^ n
```
`coe_nat` was referring to the handmade `ℕ → ℤ` coercion we had in Lean 3. This coercion didn't make it to Lean 4 (we only have the `OfNat` and `NatCast` ones), so we should start dropping it from lemma names.

Reply:
Given I've already done the modification, I don't think the present tender of "are modifying" is a particularly compelling argument any more. I'd prefer to do your suggested change as a separate PR, especially since there are `coe_nat` lemmas in other files too that also need renaming.

Reply:
I'm counting in terms of "size of the diff to master", but sure